admin/services/helpdesk.service.js

```diff
@@ -0,0 +1,20 @@
+'use strict'
+
+const roles = require('../lib/consts/service-roles.js')
+const helpdeskService = {}
+
+/**
+ * Identify if helpdesk user has not received impersonation
+ * @param {object} user
+ * @returns {boolean}
+ */
+helpdeskService.hasHelpdeskNotReceivedImpersonation = (user) => {
```

Comment:
that method name doesn't read very well.  maybe flip it to `isImpersonatingSchool()` ?

Reply:
@GuyHarwood this method has now been split to 2 different ones for readability and simplification.